README.md

```diff
@@ -1,10 +1,20 @@
 # commercetools-api-reference
 Commercetools Platform API reference documentation
 
-## Postman
+### Overview
+This repository represents the raml structure representation of commercetools APIs.
+
+### Purpose
+This public repository gives the chance to the users to have the possibility to extend our APIs and customize them.
+
+### How to use it
```

Comment:
Are you still adding more content below this?

Reply:
Yes, This is what I need to talk with @jenschude about.
I have no idea which kinds of examples are good to show to our customers